statistics/column.go

```diff
@@ -346,6 +346,13 @@ func (c *Column) dropTopN() {
 	}
 }
 
+func (c *Column) dropHist() {
+	c.Histogram.Bounds = chunk.NewChunkWithCapacity([]*types.FieldType{types.NewFieldType(mysql.TypeBlob)}, 0)
+	c.Histogram.Buckets = make([]Bucket, 0)
+	c.Histogram.scalars = make([]scalar, 0)
```

Comment:
Can we just set `Bounds`, `Buckets` and `scalars` to nil?

Reply:
This may cause panic in some Histogram functions which are hard to cover.